src/coreclr/scripts/superpmi.py

```diff
@@ -596,6 +597,10 @@ def __init__(self, coreclr_args):
         if coreclr_args.pmi:
             self.pmi_location = determine_pmi_location(coreclr_args)
             self.corerun = os.path.join(self.core_root, self.corerun_tool_name)
+            if coreclr_args.pmi_path is None:
```

Comment:
Does this need to be `if 'pmi_path' in coreclr_args:` ? Did you check what happens when `pmi_path` is not passed?

Reply:
If I don't set `-pmi_path` it is `None`. 

My understanding that the call to
```py
coreclr_args.verify(args,
                            "pmi_path",
...)
```
handle the unspecified arguments?

Reply:
Just double check because we have seen errors in the past of 'key' not part of dictionary.

Reply:
Checked and the following is behavior I am observing: 

1) when specify `--pmi` and `-pmi_path` the `PMIPATH` env variable is set
2) when specify `--pmi` but not `-pmi_path`, the variable is not set and nothing changes comparing to what was before the change
3) when specify `-pmi_path` but not `--pmi`, the warning is emitted